### DISCUSSION TITLE
For the knowhere test of GPU and CPU, is it a problem with GPU memory settings or where?

### DISCUSSION THREAD

**Discussion by Escapethebug** [milvus-io/milvus#26794] [Q&A and General discussion]
I only tested the IVF_FLAT mode. Why is there such a big difference in the test time? Is there something wrong with my settings? Video memory or core? Or does such a test fail to verify GPU performance? RTX4090 used![image](https://github.com/milvus-io/milvus/assets/45965328/065e5fc3-2826-4558-b3f1-4fd58db42c98)

**Comment by Escapethebug** [comment]
help!

**Reply by xiaofan-luan** [reply]
what test you are running?
Can you give us some test snippet you are running?

**Reply by Escapethebug** [reply]
It has been normalized, thank you very much. I am using this one that comes with https://github.com/zilliztech/knowhere/blob/66145d034f260cd7a277977ee533e8e26c76266a/tests/ut/test_search.cc#L111C3-L111C3
https://github.com/zilliztech/knowhere/blob/66145d034f260cd7a277977ee533e8e26c76266a/tests/ut/test_gpu_search.cc#L66
![image](https://github.com/milvus-io/milvus/assets/45965328/ed5e0728-f2d5-41ee-9c5b-493cd555604b)

**Comment by presburger** [comment]
@Escapethebug 

https://github.com/zilliztech/knowhere/blob/3d21950cc27e8383c9435dd2c011b7635fd44ac3/CMakeLists.txt#L52
add 89 in this line.

**Reply by Escapethebug** [reply]
It has been normalized, thank you very much.

**Comment by presburger** [comment]
@Escapethebug can you show me  nvidia-smi print out?

**Reply by Escapethebug** [reply]
It has been normalized, thank you very much.
![image](https://github.com/milvus-io/milvus/assets/45965328/8e5b95d7-8e83-4e62-9bd1-ead68d67f9d9)
